lib/src/runner/configuration/load.dart

```diff
@@ -49,13 +49,18 @@ Configuration load(String path, {bool global: false}) {
 
   if (document.value == null) return Configuration.empty;
 
-  if (document is! Map) {
-    throw new SourceSpanFormatException(
-        "The configuration must be a YAML map.", document.span, source);
+  if (document is YamlMap) {
+    var loader = new _ConfigurationLoader(document, source, global: global);
+    var config = loader.load();
+    if (config.includePath != null) {
+      var basePath = p.normalize(p.join(path, '../', config.includePath));
```

Comment:
I'd write this as `p.join(p.dirname(path), config.includePath)`.

Reply:
Wow, much better. Done.